app/src/main/java/org/fossasia/openevent/utils/SortOrder.java

```diff
@@ -0,0 +1,40 @@
+package org.fossasia.openevent.utils;
+
+import android.app.Activity;
+import android.content.SharedPreferences;
+import android.preference.PreferenceManager;
+
+import org.fossasia.openevent.dbutils.DbContract;
+
+/**
+ * User: anupam (Opticod)
+ * Date: 1/3/16
+ */
+
+public final class SortOrder {
+
+    public static final int SORT_TYPE_ID = 0;
+    public static final int SORT_TYPE_NAME = 1;
+    public static final int SORT_TYPE_ORGANISATION = 2;
+    public static final int SORT_TYPE_COUNTRY = 3;
+    private static final String PREF_SORT = "sortType";
+    private static SharedPreferences prefsSort;
+
+    private SortOrder() {
+    }
+
+    public static String sortOrderSpeaker(Activity activity) {
+        prefsSort = PreferenceManager.getDefaultSharedPreferences(activity);
+        switch (prefsSort.getInt(PREF_SORT, 0)) {
+            case SORT_TYPE_ID:
+                return DbContract.Speakers.ID;
```

Comment:
@mananwason @championswimmer , just want to confirm: we should also have one default sorting order (that of in the API) so I think of replacing Id with unsorted . Is that will be good?